entries/f/fanfiction.net.json

```diff
@@ -0,0 +1,13 @@
+{
+  "FanFiction.Net": {
+    "domain": "fanfiction.net",
+    "img": "fanfiction.net.png",
+    "contact": {
+      "twitter": "FictionPress",
+      "email": "support@fanfiction.com"
+    },
+    "keywords": [
+      "entertainment"
```

Comment:
Not sure if this is an appropriate category. Maybe `other`? :thinking:

Reply:
I consider it `entertainment` because writers release stories for people to read, which counts as entertainment to me. I can switch it to `other` if you'd like.

Reply:
Yeah I still think it should be in `other`. Sorry 😬

Reply:
No worries. I'll update it in a few.